core/src/Cabana_VerletList.hpp

```diff
@@ -530,8 +530,14 @@ class VerletList
     VerletListData<MemorySpace,LayoutTag> _data;
 
     /*!
-      \brief Given a list of particle positions and a neighborhood radius calculate
-      the neighbor list.
+      \brief Default constructor.
+    */
+    VerletList()
+    {}
```

Comment:
Not called anywhere

Reply:
True - we should technically update the code on line 427 to only create the neighbor list and then do the assignment on a separate line.